lib/plugins/aws/package/compile/events/alb/lib/validate.js

```diff
@@ -216,8 +217,36 @@ module.exports = {
     return eventAuthorizers;
   },
 
-  validateAlbAuth(auth) {
-    auth.onUnauthenticatedRequest = auth.allowUnauthenticated ? 'allow' : 'deny';
+  validateAlbAuth(auth, authName) {
+    const hasAllowUnauthenticated = _.has(auth, ['allowUnauthenticated']);
+    const hasOnUnauthenticatedRequest = _.has(auth, ['onUnauthenticatedRequest']);
+    const supportedValues = ['deny', 'allow', 'authenticate'];
+
+    if (
+      hasOnUnauthenticatedRequest &&
+      !_.includes(supportedValues, auth.onUnauthenticatedRequest)
+    ) {
+      throw new this.serverless.classes.Error(
```

Comment:
Remove this validation (this will be in scope of schema config validation which is handled here: https://github.com/serverless/serverless/issues/6562)

Here just assume that passed value is valid